core/tests/org.openjdk.jmc.common.test/src/test/java/org/openjdk/jmc/common/test/item/StreamTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * 
```

Comment:
Should this also include DataDog copyright?

Reply:
Indeed! Thanks!